### ISSUE TITLE
Multi-line function call not indented correctly; expected 4 spaces but found 0

### ISSUE THREAD

**Issue by gjsman** [squizlabs/PHP_CodeSniffer#1868]
I keep getting this error in my setup, but it has a loop issue - if I have four spaces, it tells me it wants zero. If I have zero spaces, it tells me it wants four.

![2018-01-29 1](https://user-images.githubusercontent.com/29414519/35543397-7f9b97d4-052a-11e8-8d00-58b349711ee1.png)

![2018-01-29](https://user-images.githubusercontent.com/29414519/35543378-6cb0471e-052a-11e8-91cd-1b99dff7b9ab.png)

**When I try to fix**

![2018-01-29 2](https://user-images.githubusercontent.com/29414519/35543420-9f76d1fe-052a-11e8-99e6-b950424756d7.png)

![2018-01-29 3](https://user-images.githubusercontent.com/29414519/35543432-ae78478c-052a-11e8-8930-db6d7f976d83.png)

As you can see, it tells me above it wants 4 spaces, and found none. When I fix it and add four spaces, it tells me it expects 0 spaces.

**Comment by gsherwood** [comment]
Just for the future, if you're able to post the code itself instead of (or in addition to) screenshots it makes the bug easier to replicate.

But the code is small for this one, so I've replicated it with this:
```php
<?php
Route::get('/', function () {
    return view('welcome');
}
);

Auth::routes();
```

**Comment by gsherwood** [comment]
I'm not sure what coding standard you are using, but I'm going to assume the PEAR standard as it is able to replicate those error messages.

For my sample code, I actually get 2 errors:
```
 2 | ERROR | [x] Opening parenthesis of a multi-line function call must be the last content on the line
   |       |     (PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket)
 4 | ERROR | [x] Multi-line function call not indented correctly; expected 4 spaces but found 0 (PEAR.Functions.FunctionCallSignature.Indent)
```
The first one is a important because it sets the indent levels for other sniffs. If you only fix the second error and not the first, you will end up with a conflict. If you are using a custom coding standard, make sure you either include the `PEAR.Functions.FunctionCallSignature` sniff or remove the `PEAR.WhiteSpace.ScopeClosingBrace`.

For this specific code snippet, running phpcs with the diff report, or using phpcbf to auto-fix, will show you what code the PEAR standard wants:
```diff
$ phpcs temp.php --standard=PEAR --report=diff
--- temp.php
+++ PHP_CodeSniffer
@@ -1,7 +1,8 @@
 <?php
-Route::get('/', function () {
-    return view('welcome');
-}
+Route::get(
+    '/', function () {
+        return view('welcome');
+    }
 );

 Auth::routes();
```
So it wants the code fixed like this:
```php
<?php
Route::get(
    '/', function () {
        return view('welcome');
    }
);

Auth::routes();
```
Does that make things clearer?

**Comment by gjsman** [comment]
I ran `phpcbf routes/web.php` - and now that error actually does make sense. I assumed that ` 2 | ERROR | [x] Opening parenthesis of a multi-line function call must be the last content on the line` meant the `{`, not the `(`, and so I didn't realize the conflict. Thank you! 👍

**Comment by gsherwood** [comment]
Good news. Thanks for getting back to me.
